manager/allocator/networkallocator/portallocator.go

```diff
@@ -285,7 +285,8 @@ func (pa *portAllocator) portsAllocatedInHostPublishMode(s *api.Service) bool {
 
 	if s.Spec.Endpoint != nil {
```

Comment:
@aboch: Now that I look at it, I suppose there's no reason to have the `if` at the beginning of the function. In the case where the `if` condition would be true, the function would do nothing and return `true` anyway.